doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -59,13 +59,31 @@
 <a name="3.2.2"></a>
 ### 3.2.2 Container Networking Services
 
-> This chapter should discuss networking related considerations of container network services, like:
-> * Ingress and any limitations on protocol or raw interface support
-> * Dependencies on external loadbalancers
-> * Regular service mesh related considerations
-> * Network Service Mesh related considerations
-> * Number of interfaces and IP addresses to be supported by a pod
-> * IPv6 single and dual stack needs
+Because `req.inf.ntw.01` requires the architecture to support CNI and `req.inf.ntw.16` requires the capability to attach several network interfaces to the pods the architecture must support a CNI metaplugin/CNI multiplexer.
+
+A CNI metaplugin/CNI multiplexer is capable to attach several interfaces, using different other CNI plugins, to a pod. The different network charastheristics of the interfaces, which require different networking technologies require different CNI plugins.
+
+Inter node communication required by `req.inf.ntw.02` must be served by a CNI plugin with basic networking capabilities, like Calico or Flannel.
+
+There are two types of low latency and high throughput networks required by `req.inf.ntw.04`. Network used for signalling traffic are more demanding than what an overlay network can handle, but still does not need the usage of user space networking. Due to the nature of signalling protocols used this type of networks require the NAT-less communication stated by `req.inf.ntw.03`. Due to the combination of these two requirements networks with this characteristics must be served by a CNI plugin with IPVLAN or MACVLAN support.
```

Comment:
technically SR-IOV can be also used for kernel networking

Reply:
I agree, but the true question is if we would like to use SR-IOV for kernel networking. From the debates around the usage of SR-IOV in general my conclusion is that we should rely on SR-IOV only in places where we really need to.